astropy/table/column.py

```diff
@@ -165,6 +165,17 @@ def new_like(self, cols, length, metadata_conflicts='warn', name=None):
 
         return self._parent_cls(length=length, **attrs)
 
+    def get_sortable_arrays(self):
+        """
+        Return a list of arrays which can be lexically sorted to represent
+        the order of the parent column.
+
+        For Column this is just the column itself.
+
+        :return: list of Column
```

Comment:
Like @bsipocz noted, should be something like
```
Returns
-------
arrays : list of Column
```
Though I think it is fine to just delete it too, since the title already states what it is.